recipe/meta.yaml

```diff
@@ -8,6 +8,11 @@ source:
   - url: https://altushost-swe.dl.sourceforge.net/project/nsis/NSIS%20{{ version.split(".")[0] }}/{{ version }}/nsis-{{ version }}.zip
```

Comment:
Would it work with `[win and nsis_variant == 'standard']` ?

Reply:
Not as it is right now. We always need the standard installation. The way I understand it, the log build is a "patch" of the regular build, so we need the "standard" in any case.